src/components/TokenGeneralInfo.js

```diff
@@ -31,18 +34,26 @@ const mapStateToProps = (state) => {
  */
 class TokenGeneralInfo extends React.Component {
   /**
-   * errorMessage {String} Message to show in case of error getting token info
-   * totalSupply {number} Token total supply
-   * canMint {boolean} If this token can still be minted
-   * canMelt {boolean} If this token can still be melted
-   * transactionsCount {number} Total number of transactions of this token
+   * @property {string} errorMessage Message to show in case of error getting token info
+   * @property {number} totalSupply Token total supply
+   * @property {boolean} canMint If this token can still be minted
+   * @property {boolean} canMelt If this token can still be melted
+   * @property {number} transactionsCount Total number of transactions of this token
+   * @property {boolean} alwaysShow Indicates if this token is always shown despite having zero bal.
```

Comment:
zero bal? Was it intentional, or a typo?

Reply:
I was trying to keep the line below the 100 character [line length limit](https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb-base/rules/style.js#L199).
Reworded this on [f84875e](https://github.com/HathorNetwork/hathor-wallet/pull/310/commits/f84875e9a7e8fbeea38439e8791b9d8e7ab34a69), I think it's better now.